### ISSUE TITLE
ListView Command Set extension suddently stopped working - only loads when page is refreshed with Ctrl+F5

### ISSUE THREAD

**Issue by JLRishe** [SharePoint/sp-dev-docs#9896]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Developer environment

None

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- Chrome 128.0.6613.85
- 1.13.1

### Describe the bug / error

I am the maintainer of a SPFx package that is used in a few dozen customer tenants and contains a List View Command Set extension, a Field Customizer extension, and a web part.

Two days ago, one of my customers who has been using this package for about this year reported that the List View Command Set button had suddenly stopped displaying on multiple list pages (possibly all of the list pages where it is in use).

I have met with him to troubleshoot and observed the following:

- The extension displays without issue on Library View pages, but has an issue on List View pages.
- It displays without issue in modern List View _webparts_, even for the same lists where it fails to display on their regular List View page.
- If he refreshes the page with Ctrl+F5, the Command Set button displays and works correctly. But if he then refreshes the page normally, or navigates away and returns to the page, it again fails to display. The only time it displays is when the page is loaded with Ctrl+F5.
- The Field Customizer extension from the same package is working fine, even on the pages where the Command Set is having an issue.
- The Web Part from the same package is working fine, but it is not used on list view pages.
- The browser console doesn't seem to show anything pertaining to the Command Set. There are no error messages about it (or coming from it) when it fails to load, and there is no mention of it when it loads successfully. By contrast, there are log entries indicating that the Field Customizer loaded.
- There are a few differences in the console output during a successful load vs. unsuccesful load:
  - When loading the page in such a way that the Command Set fails to load, there is a log line from deferred.js indicating "1526 install done". When loading with Ctrl+F5, there isn't.
  - There is a "Not one page nav, start prefetch" log line that appears further down in the list of console entries when loading with Ctrl+F5 vs loading normally.
- The Network tab of the browser tools shows the SPFx package's script files being retrieved differently depending on how the page is loaded.
  - When loading with Ctrl+F5 the package's script files are retrieved and the Initiator column is blank
  - When loading without Ctrl+F5, the packages script files are retrieved _twice_ - once with sp-loader-assembly as the Initiator and once with AllItems.aspx as the initiator. In both cases, the Fulfilled By column indicates `(Service Worker)`
- This issue does not currently reproduce in my tenant, and none of my other customers have reported this issue, though I'm not sure how many of them are using this Command Set on a List page since using it on a Library page is more common.

What can I do to troubleshoot or remedy this issue?

### Console when Command Set fails to load

![Not loading console markup](https://github.com/user-attachments/assets/decd0eea-a489-4122-973f-ecb9a7e5b68c)

### Console when Command Set loads

![Loading console markup](https://github.com/user-attachments/assets/7c823103-ffb3-4cfe-bd73-f58128c373d7)

### Network requests when Command Set fails to load

![image](https://github.com/user-attachments/assets/dd54adf0-5d54-4629-b298-bc2064fa15f1)

### Network request when Command Set loads

![Loading requests](https://github.com/user-attachments/assets/597dba31-b23c-40c4-b0a4-4e8d9b1f36d2)

### Steps to reproduce

Currently I only know this to reproduce in my customer's environment.

1. Navigate to a list page that has the Command Set extension configured
2. View the command bar

### Expected behavior

The Command Set button should display.

**Comment by Marker4** [comment]
I am seeing the same thing with a Command Set Extension that was developed with SPFX 1.13.1. This is a project that was working fine for years. But with the new SharePoint List UI that the tenant recently got, the buttons are no longer visible.

I tried to troubleshoot by creating a new Command Set HelloWorld Extension using the MS Learn example and spfx 1.13.1: https://learn.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/building-simple-cmdset-with-dialog-api 

When I try to gulp serve or gulp bundle, this error is thrown:
AppData\Roaming\nvm\v14.15.4\node_modules\gulp-cli\bin\gulp.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\-\-\command-extension\node_modules\@azure\logger\dist\commonjs\log.js:7:19)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  code: 'MODULE_NOT_FOUND',

If I update to spfx 1.19.0 and npm 18.19.1 to create the MS Learn Example, I am able to package and deploy without an issue. The Command Set buttons appear as expected.

Can someone run a similar test with spfx 1.13.1?

**Comment by gerkevgarderen** [comment]
We are seeing the same issue since the forced rollout of the new Lists experience.
However, even Ctrl + F5 does not consistently allow the Command Sets to load, only some of multiple tries succeed.

Same as your description we're seeing that other extensions, deployed in the same app package, are loading successfully. Even initialization logs of Command Set extensions do not show up in the console at all.

Really hope there will be a quick fix/rollback for this. It's very frustrating there's nothing we can do to rollback or fix the issue, especially since it's clear from earlier issues that there have been many lingering issues with the new Lists experience.

**Comment by SPJS** [comment]
I see the same behavior where the Command Set Extension does not load until you do a hard refresh. 

Turning off Sync under Microsoft Lists (Waffle icon in top left corner > Lists > Hover over the list and click ... >  Turn off Sync) fixes the issue, but from what I can see this setting is personal and there is no method to turn this off on a list for all users. 

It doesn't seem to be the same setting that you can find in List settings > Advanced settings > Offline Client Availability > Allow items from this list to be downloaded to offline clients. From what I can see, turning off this setting does not make any difference.

Please fix!

Alexander

**Comment by Marker4** [comment]
Can someone provide more information on this workaround? I don't see the ability to turn off syncing as described for MS Lists. 

We are using SharePoint Online. Both lists have "Allow items from this list to be downloaded to offline clients?" set to "No" which has made no difference on our end. Hard Refresh (Ctrl + F5) is also not working to get the Command Sets to display.

While it is great that there is a workaround that is working for some people, it does not appear to be working for all. It is not working on my client's tenant.

Please test with a Command Set created with SPFX 1.13.1

**Comment by SPJS** [comment]
If you click the Waffle icon (the 9 dots icon) in the top left corner of your SharePoint site you should see a "Lists" icon. Click it and you should see your list under "Recent lists". Hover over the list, click the ... icon and click "Stop syncing".

Alexander

**Comment by Marker4** [comment]
@SPJS I did try to follow your prior instructions. I'm just not seeing it on either site I am looking at:
![MSListSettingsExample](https://github.com/user-attachments/assets/cf6d6581-93a2-45dd-a043-0c524d369831)

**Comment by keraaj1980** [comment]
> @SPJS I did try to follow your prior instructions. I'm just not seeing it on either site I am looking at: ![MSListSettingsExample](https://private-user-images.githubusercontent.com/51385187/364895886-cf6d6581-93a2-45dd-a043-0c524d369831.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NjA1MDgsIm5iZiI6MTcyNTU2MDIwOCwicGF0aCI6Ii81MTM4NTE4Ny8zNjQ4OTU4ODYtY2Y2ZDY1ODEtOTNhMi00NWRkLWEwNDMtMGM1MjRkMzY5ODMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDE4MTY0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmNTM1OWEwOWZlYTY2YjRhOGJjNTUwNjRlNDJkMTIyMzM2MjFkNGM1NzU4NmE1ZmQ4NGJjYzZhNmRmZTQxZGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dKePhrIV6XbUI-M2KHCvOhghkaZsa51H3rhsPk532bQ)

Same here. I dont see the option to "Stop syncing".

**Comment by keraaj1980** [comment]
> Thanks folks. We are aware of and working on an issue where command sets don't load when a list is syncing. As mentioned in this thread, the workaround for now is to stop syncing on the list.

Workaround is not working for everyone. Please fix!

**Comment by gerkevgarderen** [comment]
> @SPJS I did try to follow your prior instructions. I'm just not seeing it on either site I am looking at:

Same here; 'Stop Syncing" is available for some lists, but for the lists where we have CommandSets deployed, this option is not available.

> It doesn't seem to be the same setting that you can find in List settings > Advanced settings > Offline Client Availability > Allow items from this list to be downloaded to offline clients. From what I can see, turning off this setting does not make any difference.

The value of this setting also does not impact the issue. I have checked sites where this was already on 'Off' for a longer time, but no luck. Sites with the setting "On" in many cases still don't show the Stop Syncing. All of the sites do not consistently load CommandSets (Ctrl+F5 sometimes causes it to render, still not consistently).

Seems to me like it's not necessarily related to the Syncing.

**Comment by reedpamsft** [comment]
@gerkevgarderen  there is another issue reported in a different thread where certain deprecated API's are not currently supported in the new UI, so you might be seeing that.

**Comment by keraaj1980** [comment]
> @gerkevgarderen there is another issue reported in a different thread where certain deprecated API's are not currently supported in the new UI, so you might be seeing that.

@reedpamsft - Do you have any updates on the fix? The workaround doesn't seem to work on my end,

**Comment by SPJS** [comment]
@Marker4 @keraaj1980 @gerkevgarderen I found an older article (from 2022 - [link](https://mikehatheway.com/2022/02/23/syncing-microsoft-lists-gotchas/)) mentioning that different “complex” field types like lookups or managed metadata will stop syncing of that list. Microsoft has most likely changed the functionality since that article, but I assume it might be something like that preventing you from finding the “stop syncing” option.

Alexander

**Comment by keraaj1980** [comment]
> @Marker4 @keraaj1980 @gerkevgarderen I found an older article (from 2022 - [link](https://mikehatheway.com/2022/02/23/syncing-microsoft-lists-gotchas/)) mentioning that different “complex” field types like lookups or managed metadata will stop syncing of that list. Microsoft has most likely changed the functionality since that article, but I assume it might be something like that preventing you from finding the “stop syncing” option.
> 
> Alexander

Thanks for sharing the link, @SPJS!

I don't have a managed metadata column or a lookup column on one of the lists and I still don't see the option to "stop sync" option.

**Comment by keraaj1980** [comment]
@reedpamsft - Do you have any updates?

**Comment by Marker4** [comment]
> @Marker4 @keraaj1980 @gerkevgarderen I found an older article (from 2022 - [link](https://mikehatheway.com/2022/02/23/syncing-microsoft-lists-gotchas/)) mentioning that different “complex” field types like lookups or managed metadata will stop syncing of that list. Microsoft has most likely changed the functionality since that article, but I assume it might be something like that preventing you from finding the “stop syncing” option.
> 
> Alexander

Just to double check this, I created a new blank list in SharePoint with just a Title field. No "Stop Sync"
![MSListSettingsExampleBlankList](https://github.com/user-attachments/assets/14c59db7-490c-4e06-aa17-34f6e1865245)

**Comment by keraaj1980** [comment]
Not sure if they rolled back the update? The list view has reverted back to the old look and the command set buttons have started appearing now.

**Comment by Marker4** [comment]
> Not sure if they rolled back the update? The list view has reverted back to the old look and the command set buttons have started appearing now.

I just did a hard refresh/clear cache and I am seeing the same thing in both lists with an issue. The old UI is back with the Views as a dropdown instead of tabs. The Command Set buttons are also back.

**Comment by reedpamsft** [comment]
@gerkevgarderen @keraaj1980 @Marker4   yes, we rolled back the new UI based on the two issues being experienced (no command customizers when a list is syncing, and lack of support for the deprecated onListViewUpdated event).

**Comment by henningeiben** [comment]
@reedpamsft: is there any release-note on the deprecation of `onListViewUpdated`? What I noticed is, that in commandbar-extensions the `IListViewCommandSetExecuteEventParameters` has a property `selectedRows` which was `undefined` in the new modern UI experience. In issue [9514](https://github.com/SharePoint/sp-dev-docs/issues/9514#issuecomment-2328922892) there was a proposed workaround - is this the official recommended future solution to use `this.context.listView.selectedRows` instead of `event.selectedRows`?

**Comment by VesaJuvonen** [comment]
@reedpamsft  and @henningeiben - just to follow up on this also from my side.

I don't think the deprecated code (`onListViewUpdated`) was never communicated externally, which seems to now cause challenges worldwide, given the millions of users for these extensions. Let's align on this internally @reedpamsft - so that we can work on the plan here. Typically any deprecation which would cause customer and partner code to break - have been communicated clearly with minimum 12 months heads up. This is required to avoid any breaking changes for deployed extensibility worldwide.

We'll work on a clear messaging this internally at Microsoft - Let's continue internal alignment on this @reedpamsft using internal channels and update the ecosystem as we have clear plan going forward. Thanks for jumping in here 👏

for the specific issues - the new UI changes have been rolled back as mentioned in the discussion.

**Comment by henningeiben** [comment]
@VesaJuvonen : I don't want to be a nitpicker, but the update did not only affected the `onListViewUpdated` method of SPFx extensions, but also the `onExecute` methode, which receives a parameter of type `IListViewCommandSetExecuteEventParameters`. This type has a property named `selectedRows`, which was `undefined` in the new modern UI ... either this was not intended or this needs to be migrated as well.

Can you confirm this?

**Comment by VesaJuvonen** [comment]
@henningeiben - no worries on the keeping us honest and you are 100% correct. All of those would be breaking changes for the public codebase. We will revisit this change and messaging internally in Microsoft. Right now the change has been reverted and you should be back to the normal situation which has worked for years. We will work alignment internally.

**Comment by buettiB** [comment]
What I experience today is: List are in old layout if our App is on the site and they are in new layout if I delete the app. This is according to what I found at: https://app.cloudscout.one/evergreen-item/mc600726/

> Users in affected tenants will see Lists feature updates as described in this blog post. These improvements will reach all Lists, except those that have been configured with the following features:
> SharePoint Framework extensions
> PowerApps forms
> Approvals
> The Playlist template
> Lists that are using these features will not receive the new user experience with this feature rollout. Future rollouts will bring support for Lists that are using these features.

**Comment by svkcode** [comment]
@reedpamsft to add to your list of things that did not work in the new UI, I noticed that after implementing the workaround, the commands showed up only in the command bar and not in the context menu, as pointed out in here: https://github.com/SharePoint/sp-dev-docs/issues/9905

And thanks for reverting back to the old UI!

**Comment by reedpamsft** [comment]
@svkcode thanks!  That is on the list.

**Comment by reedpamsft** [comment]
@buettiB right.  We had been rolling out the new UI for lists with command customizers, but given the scope of the issues reported here we suspended that rollout.  Since the issue was only with command customizers, lists without command customizers should continue to see the new UI.

**Comment by AKCprcrn** [comment]
I'm still experiencing this issue even after the rollback. Have to press ctrl+F5 each time I want to see the list customizer. Some users report ctrl+F5 doesn't help anymore.

**Comment by reedpamsft** [comment]
@AKCprcrn are you seeing the new UI (with views in tabs across the top) or the old one (with the views as a dropdown)?

**Comment by JonEOffice** [comment]
@reedpamsft  just for your info, last week (thursday) we had all users get the new version of the list, but now with our custom buttons being loading. We had the issue that we were using event.selectedrow so the selected buttons didn't show up. But the other buttons were showing up and working. This was rather unexpected and strange to suddenly get the new UI. CTRL+F5 and the fix in regards with adding stuff to the end of the URL didn't work anymore to get it back to the old version.
Today we are all back into the old UI again. So my guess is there was a hickup last thursday.

**Comment by reedpamsft** [comment]
Thanks for confirming, @JonEOffice .  We did indeed start rollout of support for command customizers in the new UI, but halted once it became clear that there was a large class of customizers which needed a particular event and data object which the new UI missed.

**Comment by AKCprcrn** [comment]
> @AKCprcrn are you seeing the new UI (with views in tabs across the top) or the old one (with the views as a dropdown)?

No, it's an old UI, with views in a dropdown at the right top. When its failing to load, I'm getting these and similar messages in the console output:

```
All Entries.aspx:59  A preload for '***-command-set_9fc6d66485a74ded547e.js' is found, but is not used because the request credentials mode does not match. Consider taking a look at crossorigin attribute.

All Entries.aspx:59  A preload for '***/ControlStrings_en-us_b4cd8c7b74220c708c8044df7f813da4.js' is found, but is not used due to an unknown reason.

Access to fetch at '***/ControlStrings_en-us_b4cd8c7b74220c708c8044df7f813da4.js' from origin '***' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

The FetchEvent for "***/ControlStrings_en-us_b4cd8c7b74220c708c8044df7f813da4.js" resulted in a network error response: the promise was rejected.

```

**Comment by SPJS** [comment]
I'm seeing the "Old UI" on a list, but when refreshing the page the command set buttons in the banner disappears - still showing the Old UI and the command set does not load. If I do a hard refresh (Ctrl + Shift + r) it reappears.

 If I disable sync (Waffle icon in the top left corner > Lists > Hover over the list in recent lists > Click ... > Stop sync) it works every time and does not disappear on reload of the page.

Checking /_vti_pvt/service.cnf shows this version:
vti_encoding:SR|utf8-nl
vti_extenderversion:SR|16.0.0.25221

When the command set does not load I see this in the developer console in Chrome: 
![image](https://github.com/user-attachments/assets/7b6bbc80-af4e-4b84-a61b-85c878b3e04c)

Alexander

**Comment by reedpamsft** [comment]
@AKCprcrn thanks for the details.  This seems like a new issue.

**Comment by Dangerous-Mind** [comment]
Is this still an issue?
My command sets that don't appear on pages with ?env=WebView, But they work fine on ?env=Embedded.
The libraries still use the old design.
SPFx version 1.19

**Comment by reedpamsft** [comment]
@Dangerous-Mind turning off sync is currently still required for command set customizers to work, particularly if you're seeing the old design.

**Comment by Dangerous-Mind** [comment]
@reedpamsft 
It's working perfectly on any of the libraries with Offline Client Availability On.
Except if I'm using ?env=WebView.

**Comment by reedpamsft** [comment]
@Dangerous-Mind interesting; would you mind sharing the customizer so we can have a look?  Feel free to remove any business logic/etc., as long as the end behavior (working vs. not working) is still the same.

**Comment by Dangerous-Mind** [comment]
The code was a real mess. It was in SPFx version 1.13 and I'm upgrading it to version 1.19 but still have some work to do in it.
It would take some time to scrap the business part of the solution.

Is this enough?

```JSON
"dependencies": {
    "@fluentui/react": "8.106.4",
    "@material-ui/core": "^4.12.4",
    "@material-ui/icons": "^4.11.3",
    "@material-ui/lab": "^4.0.0-alpha.61",
    "@microsoft/decorators": "1.19.0",
    "@microsoft/sp-adaptive-card-extension-base": "1.19.0",
    "@microsoft/sp-core-library": "1.19.0",
    "@microsoft/sp-dialog": "1.19.0",
    "@microsoft/sp-listview-extensibility": "1.19.0",
    "@pnp/graph": "^3.15.0",
    "@pnp/sp": "^3.15.0",
    "@pnp/spfx-controls-react": "3.14.0",
    "flux": "^4.0.4",
    "pizzip": "^3.1.4",
    "react": "^17.0.1",
    "react-dom": "^17.0.1",
    "tslib": "2.3.1"
  },
```

Manifest
```JSON
{
  "$schema": "https://developer.microsoft.com/json-schemas/spfx/command-set-extension-manifest.schema.json",
  "id": "e594f379-76e5-45b7-afed-b5c2a9cdbd0c",
  "alias": "ListActionsCommandSet",
  "componentType": "Extension",
  "extensionType": "ListViewCommandSet",
  // The "*" signifies that the version should be taken from the package.json
  "version": "*",
  "manifestVersion": 2,
  // If true, the component can only be installed on sites where Custom Script is allowed.
  // Components that allow authors to embed arbitrary script code should set this to true.
  // https://support.office.com/en-us/article/Turn-scripting-capabilities-on-or-off-1f2c515f-5d7e-448a-9fd7-835da935584f
  "requiresCustomScript": false,
  "items": {
    "UpdateMetadataCommand": {
      "title": {
        "default": "Update Metadata"
      },
      "iconImageUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iaXNvLTg4NTktMSI/Pgo8IS0tIEdlbmVyYXRvcjogQWRvYmUgSWxsdXN0cmF0b3IgMTYuMC4wLCBTVkcgRXhwb3J0IFBsdWctSW4gLiBTVkcgVmVyc2lvbjogNi4wMCBCdWlsZCAwKSAgLS0+CjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+CjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0iQ2FwYV8xIiB4PSIwcHgiIHk9IjBweCIgd2lkdGg9IjUxMnB4IiBoZWlnaHQ9IjUxMnB4IiB2aWV3Qm94PSIwIDAgNDg3LjIzIDQ4Ny4yMyIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgNDg3LjIzIDQ4Ny4yMzsiIHhtbDpzcGFjZT0icHJlc2VydmUiPgo8Zz4KCTxnPgoJCTxwYXRoIGQ9Ik01NS4zMjMsMjAzLjY0MWMxNS42NjQsMCwyOS44MTMtOS40MDUsMzUuODcyLTIzLjg1NGMyNS4wMTctNTkuNjA0LDgzLjg0Mi0xMDEuNjEsMTUyLjQyLTEwMS42MSAgICBjMzcuNzk3LDAsNzIuNDQ5LDEyLjk1NSwxMDAuMjMsMzQuNDQybC0yMS43NzUsMy4zNzFjLTcuNDM4LDEuMTUzLTEzLjIyNCw3LjA1NC0xNC4yMzIsMTQuNTEyICAgIGMtMS4wMSw3LjQ1NCwzLjAwOCwxNC42ODYsOS44NjcsMTcuNzY4bDExOS43NDYsNTMuODcyYzUuMjQ5LDIuMzU3LDExLjMzLDEuOTA0LDE2LjE2OC0xLjIwNSAgICBjNC44My0zLjExNCw3Ljc2NC04LjQ1OCw3Ljc5Ni0xNC4yMDhsMC42MjEtMTMxLjk0M2MwLjA0Mi03LjUwNi00Ljg1MS0xNC4xNDQtMTIuMDI0LTE2LjMzMiAgICBjLTcuMTg1LTIuMTg4LTE0Ljk0NywwLjU4OS0xOS4xMDQsNi44MzdsLTE2LjUwNSwyNC44MDVDMzcwLjM5OCwyNi43NzgsMzEwLjEsMCwyNDMuNjE1LDBDMTQyLjgwNiwwLDU2LjEzMyw2MS41NjIsMTkuMTY3LDE0OS4wNiAgICBjLTUuMTM0LDEyLjEyOC0zLjg0LDI2LjAxNSwzLjQyOSwzNi45ODdDMjkuODY1LDE5Ny4wMjMsNDIuMTUyLDIwMy42NDEsNTUuMzIzLDIwMy42NDF6IiBmaWxsPSIjMDIzODZhIi8+CgkJPHBhdGggZD0iTTQ2NC42MzUsMzAxLjE4NGMtNy4yNy0xMC45NzctMTkuNTU4LTE3LjU5NC0zMi43MjgtMTcuNTk0Yy0xNS42NjQsMC0yOS44MTMsOS40MDUtMzUuODcyLDIzLjg1NCAgICBjLTI1LjAxOCw1OS42MDQtODMuODQzLDEwMS42MS0xNTIuNDIsMTAxLjYxYy0zNy43OTgsMC03Mi40NS0xMi45NTUtMTAwLjIzMi0zNC40NDJsMjEuNzc2LTMuMzY5ICAgIGM3LjQzNy0xLjE1MywxMy4yMjMtNy4wNTUsMTQuMjMzLTE0LjUxNGMxLjAwOS03LjQ1My0zLjAwOC0xNC42ODYtOS44NjctMTcuNzY4TDQ5Ljc3OSwyODUuMDg5ICAgIGMtNS4yNS0yLjM1Ni0xMS4zMy0xLjkwNS0xNi4xNjksMS4yMDVjLTQuODI5LDMuMTE0LTcuNzY0LDguNDU4LTcuNzk1LDE0LjIwN2wtMC42MjIsMTMxLjk0MyAgICBjLTAuMDQyLDcuNTA2LDQuODUsMTQuMTQ0LDEyLjAyNCwxNi4zMzJjNy4xODUsMi4xODgsMTQuOTQ4LTAuNTksMTkuMTA0LTYuODM5bDE2LjUwNS0yNC44MDUgICAgYzQ0LjAwNCw0My4zMiwxMDQuMzAzLDcwLjA5OCwxNzAuNzg4LDcwLjA5OGMxMDAuODExLDAsMTg3LjQ4MS02MS41NjEsMjI0LjQ0Ni0xNDkuMDU5ICAgIEM0NzMuMTk3LDMyNi4wNDMsNDcxLjkwMywzMTIuMTU3LDQ2NC42MzUsMzAxLjE4NHoiIGZpbGw9IiMwMjM4NmEiLz4KCTwvZz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8L3N2Zz4K",
      "type": "command"
    },
    "GoToIndexCommand": {
      "title": {
        "default": "Go to Index"
      },
      "iconImageUrl": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAYAAAD0eNT6AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAOxAAADsQBlSsOGwAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAACAASURBVHic7d17tF11dejx7zkhJITwEPJEBSyipOERDFQpqGiRhyBKi1b0Gp+l3GsVlNqWjnE76h23HdhbH9DrRUUvhaKgXFAICGgELERAXuEVRB4KEhICAgHCIwk594/fOeSwOfuc/VhrzfX4fsaYg8AI2XPvs3fW3PM31+8HkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRJkiRNbCA6AUkd2x04FNgbeB0we/i/PwzcB1wP/GT415IkqcI2Bz4F3A4MdRhXA0djgS9JUiUdCNxF5xf+1lgK7Fl00pIkqTdTgJOBF+n94j8SzwOfKDZ9SZLUrfnArfR/4W+Nk4p8EpIkqTMDwKeBZ8n+4j8Sxxb2bCRJ0oRmAheR34V/9HLAXgU9J0mSNI5DgZXkf/EfiWXAYCHPTJIkvUKWg37dxjEFPD9JktQir0G/TuPm/J+iJEkabRGwlriL/0jsnvcTlSRJMAtYTPyFfyT+Ot+nK0mSDgJWEH/RHx0/yPUZS5LUYFOBrwEbib/gt8ayHJ+3JEmNNQ+4hfgLfbv4XX5PXZKk5iliR78s4vG8XgBJkpqmqB39LAAkSSqJMg76WQBIkpSTyB39LAAkSQpQ9kE/CwBJkjJWlh39LAAkSSpAlQb9LAAkScpA1Qb9LAAkSepDVQf9LAAkSepRlQf9LAAkSepB1Qf9LAAkSepCXQb9LAAkSepQnQb9LAAkSZpAHQf9LAAkSRpHXQf9LAAkSWqjzoN+FgCSJLVowqCfBYAkSaM0ZdDPAkCSJJo36GcBIElqvCYO+lkASJIaramDfhYAkqRGavqgnwWAJKlxHPSzAJAkNYiDfhYAkjqwWXQCUobmAd8DFkQnIkllNxidgJSRRcCNePGXJKkRHPRzCUBSD1wCUJUdBJwJ7BCdiCRVjUsAqqKRQb/L8eIvST2xA6CqcdBPkjJgB0BV4qCfJGXEDoCqYCbwHeA90YlIUl1YAKjsHPSTpBy4BKCyctBPknJkB0Bl5KCfJOXMDoDKxkE/SSqAHQCVhYN+klQgCwCVgYN+klQwlwAUyUE/SQpiB0BRHPSTpEB2ABTBQT9JCmYHQEVy0E+SSsICQEVx0E+SSsQlAOXNQT9JKiE7AMqTg36SVFJ2AJQXB/0kqcTsAChrDvpJUgVYAChLDvpJUkW4BKAsOOgnSRVjB0D9ctBPkirIDoD64aCfJFWUHQD1wkE/Sao4CwB1y0E/SaoBlwDUKQf9JKlG7ACoEw76SVLN2AHQRBz0k6QasgOgdhz0k6QaswDQWBz0k6SacwlAoznoJ0kNYQdAIxz0k6QGsQMgcNBPkhrHDkCzOegnSQ1lAdBcDvpJUoO5BNA8DvpJkuwANIyDfpIkwA5AkzjoJ0l6iR2A+nPQT5L0ChYA9eagnyRpTC4B1JODfpKkcdkBqB8H/SRJE7IDUC8O+kmSOmIHoB4c9JMkdcUCoPoc9JMkdc0lgOpy0E+S1DM7ANXkoJ8kqS92AKrHQT9JUt/sAFSHg36SpMxYAFSDg36SpEy5BFBuDvpJknJhB6C8HPSTJOXGDkA5OegnScqVHYBycdBPklSISdEJ6CWHktb6F0YnokYYGI6dgVeRuoFrgRcDc5JUoIHoBMRU0qDfZ/HnoVhDwCrgweH4HXA/cDfw6+F/HwrLTlKmvODEctBPVbIOeAhYDtxJKg6WA7cBTwXmJakHFgBxFgGnAdOiE5EysBK4aTjuJBUGy7FjIJWWBUDxZpEG/Y6ITkTK2ePALcCtwLLhXy8HNkYmJSmxACiWO/qp6Z4hFQTXAEuBXwC/D81IaigLgGJMAb4IfAH3XpBa3U8qBkaKgruwSyDlzgIgf/NJg357RiciVcRjwJXAz4bj3th0pHqyAMiXg35S/1aSugNLgEtJtyNK6pMFQD4c9JPyMUS6y2CkO3AFaQMjSV2yAMieg35ScZ4ndQcuBs4n7VMgqQMWANlx0E+KtxxYTCoIluI+BFJbFgDZcNBPKp8HSYXAhcDPgRdi05HKxQKgfw76SeW3BrgIOA+4DFgfm44UzwKgdx7dK1XT48AlwFmkIUL3HFAjWQD0xkE/qR4eAi4gdQacGVCjWAB0x6N7pfq6BziX1Blw8yHVnhexznl0r9QcNwHfAs4Gng3ORcqFBUBnHPSTmmkN8H3gG6TTDKXasAAYn4N+kkbcBPzHcDwenIvUNwuA9g4FzgDmRCciqVSeAX4AnA5cF5yL1DMLgFdyRz9JnboZOAU4B/cWUMVYALycg36SerEK+Cbwv0nHGUulZwGwiYN+kvr1Aml54F+AO4JzkcZlAeCgn6R8LAW+RDqPwA2GVDpNLwAc9JOUt+XAV0h3D6wLzkV6yaToBIJMAf4J+DqwVXAukuptJnAk8FFSJ+A2YENoRhLN7AA46Ccp0iPAV4FTgeeCc1GDNa0DsIh0JOhroxOR1FjTSQeKHQtsSbqV8IXQjNRITekAOOgnqaweIy1HfpW09bBUiCZ0AA4FLgcWRiciSWOYBhwI/AUwGViGw4IqQJ0LAAf9JFXJNOCdpKWBAeBG4MXQjFRrdV0CcNBPUtU9APwD6fZB9xFQ5uq41/0iUuXsxV9Sle0EnAlcD7w9OBfVUJ06AA76SaqzJcCJpH0EpL7VpQA4iFQp7xCdiCTlaCPwXeBvgZXBuajiqj4EOBX4X6RBv62Dc5GkvA0AewHHkQadf4nHEKtHVe4AOOgnqelWAH8PnBWdiKqnqh0Ad/STpNT5PArYB7gOeDI2HVVJ1QqAmcC5pPWvycG5SFJZvIG0f8Bk4FrcP0AdqNISgIN+kjSxe4HPAJdFJ6Jyq8I+AFOAk0nb+Xrxl6TxvR64FFgM7Bici0qs7EsA80hv5KOpVrdCkqKNLAtsDvwClwXUoqwX1QHgv5Fu8dsiOBdJqrplwCdJRw9LQDk7ADOBc4DP4aCfJGVhDqkA2BK4GrsBonwdAAf9JClf9wKfAn4enYhilaUD4I5+klSM7YCPANuQugEbYtNRlDJ0ANzRT5Ji3MWmE1TVMNEdAHf0k6Q4M4FPkGYDfk46bEgNEdUB8OheSSqXX5K+lN0dnYiKEdEBOJS0qc/CgMeWJI3t1cDHgdV4u2AjFLkT4MiOfpeQbkmRJJXLlsDpwPnAq4JzUc6KWgJw0E+SquVB4MPANdGJKB9FLAE46CdJ1bMN6e/vSaTbBYdi01HW8uwAOOgnSfVwJfAhYFV0IspOXgWAO/pJUr2sAP4cWBqdiLKR9RLAFOCfgNNwRz9JqpOtSUsC60inC6risuwAOOgnSc1wIfAx4MngPNSHrAqARaRv/dMy+vMkSeV2D/A+YHl0IupNv/sAzCRN+J+JF39JapJdgetIRYAqqJ8ZgINIO/rtk1EukqRqmUIaDNwCuAJvFayUXpYApgBfBL5AsTsJSpLKazHwX4CnohNRZ7otAOaTBv32zCEXSVK1LSPt/fJQdCKaWDff4BeRTovy4i9JGssC0nVi3+hENLFOCgAH/SRJnZoL/CfwwehENL6JhgAd9JMkdWsz4M9Iy8xXxaaidtoVAJOBLwNfxx39JEndGwAOBF4DXApsDM1GrzDWEOAM4ALgrQXnIkmqp4tJtws+G52INmktALYFlgALA3KRJNXXDcARwOroRJQMtvz6+3jxlyRlb1/gWtIOgiqB0QXAScDBUYlIkmrvD0h3COwdnYg2LQHsBNxF2s5RkqQ8rSEtB1wTnUiTjXQATsKLvySpGNuQbjE/NDqRJhsgDf6twE1+JEnFWgd8CDg/OpEmGgQOx4u/JKl4m5OGzz8RnUgTDZJ2+5MkKcIk4NvAp6MTaZpBYK/oJCRJjTYA/BtwfHQiTTJIugNAkqRIA8DXgL+LTqQpBkhDGJOjE5Ekadjnga9GJ1F3g8D66CQkSRrlK6Tb05WjQeDx6CQkSWrxz8DnopOos0nAYcAu0YlIktTiYOAR4MboROpoEF9YSVI5DQBfBz4SnUgdDQKLo5OQJKmNQeAM4M+jE6mbAdKLexfwhuBcJElqZx1wFPDj6ETqYhIwRLoT4IjgXCRJamcS8KekEwQfCM6lFkaOA94MuBnYIzAXSZIm8hTwdmBZdCJVN3Ic8AbgWNwTQJJUblsDlwA7B+dReYOjfn0d3nMpSSq/HYDLgBnRiVTZpJZ/v4HUBfiTgFwkSerUDNJSwDnYve5JawEAcDXwW+BQ0myAJEll9Bpgb+D7wMbgXCpnrAIA4FbgR8ABwOzi0pEkqSu7Aq8GLopOpGraFQAAjwL/Thq4+CM23TEgSVKZvAl4Brg2OpEq6fSifgipGJiTXyqSJPVsI3A08MPoRKqim2/1s4H/C7w7p1wkSerHc8A7SXe1aQKDE/+WlzxC2i3wBOCFfNKRJKlnW5A6ADtFJ1IFva7rzwe+B+yZYS6SJGXhVmB/YG10ImU23hDgeBwQlCSV1RxgN+C86ETKrNcCANL2wZeR1loOAqZnkpEkSf2bRzpB8OroRMoqq2/uDghKkspmI/A+YHF0ImWUZet+APgs8CVgSoZ/riRJvXoa2A+4MzqRsslj7X4BaUBwXg5/tiRJ3fo1aV5tTXQiZdLPDEA7q4Dv4ICgJKkctgf+kHRmgIblUQCAA4KSpHJ5I2k5wO2ChxXx7dwBQUlSGWwg7RTonQF0txNgr9xBUJJUBpsBPwDmRidSBkWvz7uDoCQp2pXAu4AXoxOJlNcMQDuPAmeRBjL2KfixJUkCeB3pC/CV0YlEipzQ94hhSVKUjaQuwBXRiUSJvkVvFnAGDghKkoq3AtgL+H10IhGKXgJotRY4B3gCeAdpQEOSpCJsTZpNOyc6kQjRBcCI60lnOB9Aum1QkqQivIF0t9qN0YkULXoJoNVU0lkCn6F8uUmS6ul54M3AbdGJFKmsF9n3At8GZkQnIklqhNuBfWnQfjVlWQJodTdwJmltZtfgXCRJ9Tcb2BxYEp1IUcpaAIADgpKkYu0HXAU8EJxHIcq6BNDKHQQlSUX4DenWwKejE8lbmTsAoz1K2jTII4YlSXl6FbAtcEl0Inmr4oXUHQQlSXkaAg4HLo1OJE9VLAAgneR0JmkbR0mSsvYQsDuwJjqRvFRlCaDVM8DZOCAoScrH1qSD6xZHJ5KXqnYARnNAUJKUhyHgYGp6a+BgdAIZuJO0g9OppB+WJElZGAC+BWwZnUgeqroE0GoDcBlwHXAQMD02HUlSTbyKtMxcuy5AHZYAWnnEsCQpSxuAtwA3RSeSpTosAbRaDRwB/DUN2tNZkpSbzYDTqdnAeV2WAMZyLR4xLEnKxlzSLYHXRieSlTouAbTyiGFJUhaeBuYBK6ITyUKdOwAjHBCUJGVhCvBq4P9FJ5KFpn0jdkBQktSvw0hfLCutaQUApOf8WdKywJTgXCRJ1XMPsAcVHzRvwhLAWK7HAUFJUm+2B54Dro5OpB9N7ACMNg34MnBcdCKSpEp5ljQQ+GB0Ir1qagdgxHrSmc8OCEqSujGZdCz9+dGJ9KrpHYDRHBCUJHVjCHgbcE10Ir2o406AvRrZQfAEKj7YIUkqxABwChW9ljZ9CWAsDghKkjo1F7gPuC06kW65BNCeOwhKkjqxAngjsDY6kW7YAWjPHQQlSZ3YmjRUflVwHl3xm21n5gJnAu+KTkSSVErPArsCD0cn0ik7AJ15BjgbeAJ4BzU7ElKS1LfJwFbAxdGJdMoOQPfmA98D9oxORJJUKi+Stgi+KzqRTtgB6N6jwL+T1nz+CIsoSVIySLp77LzoRDrhxas/h5CKgTnBeUiSymEI+GPSAHmpVXLzghK5HNgL+HF0IpKkUhgATo5OohMuAfRvLXAODghKkpKdgV8A9wfnMS6XALK1gDQgOC86EUlSqBtJc2JD0Ym0YwcgW6uA7+CAoCQ13Q6kIuDX0Ym04wUqPw4ISlKz3QzsQ0m7AA4B5scBQUlqtjcBh0cn0Y5LAPlyQFCSmm034PToJMbiEkBx3EFQkprpPZRwi2A7AMVxB0FJaqZSdgEsAIo1csTwLaSTBafFpiNJKsBc4AbgnuhERvNbaJxZwBnAu6MTkSTl7nrgLdFJjGYHII4DgpLUHK8BfgY8GJ3ICG8DjDUEnAIsBG4LzkWSlK+/jU5gNJcAymMq8CXgM/hzkaQ6GiLdCXZHdCLghaaMjiRtJzwjOhHV2rPA54BtgemjYltgK9LdKtsPx3bYLZSy8h/AougkwAKgrOYCZ5LuFJDy8ATpwt6p7UfFDNIW1zsAs4f/OYu0xjkb2DzTTKV6WQ/sCjwQnYgFQHkNkL6h/TMwJTgX1U+3BUCnBknFwc7AjsOx0/A/dwH+AN/P0qnA8dFJWACUn0cMKw95FQATGQReC7y+JeaTigPvTFITrCV9Dp6ITMICoBqmAV8GjotORLURVQCMZwqp0J0H7D78zz1JhYF/V6luvgD8a2QCfqiqxSOGlZUyFgDtbEU6WfMPSZ2ChcMxNTIpqU8PkLpfG6ISsACoHncQVBaqVACMZTPgjcD+wAGkgmAe/p2majkaOD/qwf2wVJMDgupX1QuAscwgHbS1P/BWYF/sEqjcrgbeFvXgFgDV5oCgelXHAqDVFFIR8DZSl+AA0nKCVCYLgZsjHtgCoPrcQVC9aEIB0GoSqWg+aDjeih00xTsL+GjEA3vBqA93EFQ3mlgAtJoOvB04DDiCtF+BVLQXSO+9R4p+YAuAenFAUJ2yAHil+aRC4D3Afrj9sYpzEnBy0Q9qAVA/A8BnScsCtjfVjgXA+GayqTNwGKlbIOXlN6RbAjcW+aAWAPXlgKDGYwHQuWmkczneQyoIZsemo5o6DLgsOgnVx1TgFFJVOWQYo+Jx1ItB0t0E3wTWEP9zNOoTF1AwOwDN4A6CamUHoH9TSZ2BjwDvxVMQ1Z8NpEO0VhT1gA65NMPlpK1UfxydiFQjzwOLgQ+QlgU+CiwhfZuTurUZ8MkiH9AOQLO4g6BG2AHIz87Ah4APk84vkDr1O+B1wIvRiai+FgDLiV/zMuLCGYBi7A98l9QtiP6ZG9WIIyiISwDNtAx4E3Aq6Q0nKR9LSZ2AOcAJpNu9pPF8rKgHcglADgg2k0sAMQaBdwLHAkeR1n2l0dYBrwYey/uB7ADIAUGpOBtJg4IfIM0KfBFYHZmQSmdz4IPRSahZBoDP41plU8IZgPKYAiwCbif+fWGUI25ACjAfuJX4D4CRb1gAlM8AabfBpcS/P4z42IOcuQSgVncCb8YBQaloQ6R9BfYnHVV8MX4Gm+wj0Qmo2Q4BVhJfCRvZhx2AatiDdF78euLfM0axsYqch0TtAGg8l5NuF/xpdCJSQ91Omg/YDfgGaUZHzTAbODjPB7AA0ERWkjoBJwIvBOciNdV9wH8FdgFOI3UEVH/HRCcgjXBAsD7hEkC17UQ6kXAD8e8lI794CtiCnNgBUDccEJTK4QHgL0l7eJwXnIvysxVweHQSUisHBKsddgDqZT/gKuLfV0b2YYGnUpoFXEL8B8ToPiwA6ukg4Bbi319GdvEcsA05cAlA/VhNOrnKAUGpHJYAC0nnyrvFcD1MJW0QJZWWA4LVCjsA9bctcDKpOI9+vxn9xcVIJTcVOIV04En0B8YYPywAmmM34CfEv+eM3mMdsH3rD7ZfLgEoS88DxwOHkXaxkhTvV6QNZY4EHgzORb2ZTPr5ZcoCQHnwiGGpfBYD80hHEDuzUz1HRScgdcMjhssbLgE0226kLb6j34dG5/EcMH2sH6ZUZg4Ili8sAATwfuD3xL8fjc7i6LF/jL1xCUBFcAdBqZzOA3YHfhSdiDryvugEpH64g2A5wg6AWr0feIz496bRPp4ENm/3A+yWHQAVzQFBqZzOIy3XXRCdiNraBjgwqz/MAkARRnYQPAGnkaUyeQT4M+ADpG6AysdlANXGAmA58a21poVLAJrIbOBC4t+rxstjBekOK6kW3EGw+LAAUKcWAWuJf88am2LvcX9iHXIJQGXgDoJSeZ0F7AvcHp2IXvLuLP4QCwCViQOCUjktZ9OtvIp3WHQCUl4GSB0BdxDML1wCUK/+lPT+iX4PNzk2kMPhQFKZuINgfmEBoH7sCFxD/Pu4yXHMhD+lCbgEoDK7E9gP+EZ0IpJe5kHgHcC/kC5GKl4mcwBSFbiDYLZhB0BZORJYQ/x7umnxKH6JV4PMAi4h/oNXh7AAUJb2AO4j/n3dtHhzJz+cdqweVCUjOwieiDsISmVyO+lWwZ9GJ9Iw3g2gRnJAsL+wA6A8TAJOxk29ioqrO/uxSPXjDoK9hwWA8vRB3D2wiFgPbNXhz0SqpSNJAzHRH8YqhQWA8rY38Fvi3+t1j57vBnAGQHVwEWlJwB0EpfK4BXgLcFN0IjX3zugEpDJwB8HOww6AirIlcDHx7/m6xi2d/yik+nNAcOKwAFCRJgGnEf++r2NsBGZ2/qOQ6s8BwfHDAkARjsfPZB7x/m5+CFJTuIPg2GEBoCiLgHXEfwbqFG6XLrUxF/gJ8R/SMoUFgCIdDDxF/OegLnFPdy+/1CwDwOdxQHAkLAAUbQHwCPGfhbrE3O5efql5HBBMYQGgMngj8BDxn4c6xNFdvvbuA6DGuZN0gMappA+NpDh3AwcA90cnUgMHRCcgVUmTBwTtAKhMdiStY0d/LqocN3b9qksN19Qjhi0AVDZzSKcKRn82qhob8FwAqWtN3EHQAkBlNAtYRvzno6rxrm5ebGcApPTBOQVYCNwWnIvUZKuBA4Hrg/OoKucApD5MoxlbltoBUJltSzpEKPpzUrW4opcXW9LL1f2IYQsAld0M0l070Z+VKsVaYHIvL7akl6vzgKAFgKpgFvAr4j8vVYqFnb64zgBI7a0GjgBOAF4IzkVqotWkwbbfBudRJft2+hstAKTxDeGAoBTpd6QiYGV0IhXRcQEgqXN1OmLYJQBVzR7AY8R/dsoet/f6AkuaWB0GBC0AVEX7AmuI//yUOdwQSMpZ1QcELQBUVQeSZnKiP0NljgM7eSGdAZB644CgFOMq4OOkC53G5hyAVJAqHjFsB0BV94/Ef47KGj/o/WWV1K2qDQhaAKjqBoAziP8slTF+2/vLKqlXVRkQtABQHUwGfkL856mMMauP11VSj+ZS/r+UHs3t2UvF2prqLcEVEQdP9MI5BChlbyVwCHAi5R0QfDI6ASkjTwGHAyuiEymZvSb6DRYAUj6GgK8AbwHuCs5lLKujE5Ay9BDwXuC56ERKxAJACrYMeBNwKqkoKIuboxOQMnYT8BfRSZTIgugEJG1yJOmbd/Ta4BDw4ZyfqxTlK8R/vsoQ60l3J0kqiTLsIPgcsE3eT1QKMgm4lPgLcBlinz5fS0kZGwA+DzxPzF8KZ+f/FKVQM4DfEH8Bjo5P9vtCSspHxA6CG4B5RTw5KdiewDPEX4Qj49S+X0VJudkCOA3/QpDy8EGqsztnHvGf/b+EkvJWxIDgHaSCQ2qSrxF/IY6KJzJ4/SQVIM8BwRXA64p7KlJpTAauI/5iHBVz+38JJRUhjwHBB0jzBlJTvR5YQ/zFOCL+JIPXT1KBFgC30/+H/2d4KIgEcAzxF+OI+KssXjxJxZoMfIG0jtfth/5h4GOkjoKk5HTiL8hFx9czeeUkhZgOfBq4mrS7V7sP+rOkUwg/DGwekqlUbtNIw7DRF+Ui44p2L4bfDqRq2QbYA9gFmEL6DD8N3E/aU8DDUKTxzQd+SSoGmmAVDgJKkgTAccR/My8ytsvmZZMkqdoGgB8Tf2EuKv54rBfB44AlSU0zBHyK5myUM+b23xYAkqQmehg4ITqJguw21n+0AJAkNdVZwPnRSRRgl+gEJEkqm5nAI8Sv0+cZt2b2akmSVCPvI/4inWesxdv+JUka07nEX6jzDPcCkCRpDLOBx4m/UOcVB7Q+YYcAJUlKcwAnRSeRo9e3/gcLAEmSktOBpdFJ5OQVdwJYAEiSlGwE/pJ06FbdWABIkjSOO4F/jU4iB+4FIEnSBLYA7iN+cC/LeCzTV0iSpJo6hPiLdtYxPdNXSJKkmvoh8RftLONlhwI5AyBJ0thOBF6ITiJDO47+FwsASZLGdj9wSnQSGXrt6H+xAJAkqb3/CayMTiIjFgCSJHXoaeAfopPIyI4T/xZJkjRiELiR+CG+fmNJ65OSJEntbQSOJ11Eq8wOgCRJPfgR8d/i+4lngYHMXxVJkmpud+BF4i/k/cTMkSfjEoAkSZ25AzgnOok+vWbkFxYAkiR17r8D66KT6MOckV9YAEiS1LnfAGdEJ9EHCwBJknr0P0gDdVVkASBJUo8eBv5PdBI9mj3yCwsASZK69yXgmegkemAHQJKkPjwGfDM6iR7Mnvi3SJKk8cwhzQJE39vfTdw1krwdAEmSerMKODM6iS7Nmfi3SJKkiexI2hcg+pt9N7EF2AGQJKkfD1K93QGdA5AkKQO7Ua0zAvYBOwCSJPXrV6STAqtie7AAkCQpC1+NTqAL24EFgCRJWbgGuCE6iQ5ZAEiSlKF/i06gQxYAkiRl6FxgRXQSHbAAkCQpQ+uBb0Un0QELAEmSMnYa8Hx0EhOwAJAkKWOPkpYCyszbACVJykHZTwncLjoBSZLqahnxO/61i9VgB0CSpDycEZ3AOLYFGIjOQpKkGtqOdEvg1OhE2phqB0CSpOw9DlwQncQ4trIAkCQpH9+OTmAcFgCSJOXkKuDe6CTasACQJCknQ8B3o5NowwJAkqQcnU0qBMrGAkCSpBzdC9wYncQYLAAkScrZOdEJjMECQJKknJ0LvBidRAsLAEmScraSdEdAmVgASJJUgO9FJ9BiugWAJEn5uxDYEJ3EKFMsACRJyt/vgWuikxjFswAkSSrIhdEJjGIBIElSQX5IeTYFcglAkqSCPADcGp3EMDsAkiQVqCzL4rs/EAAAAgBJREFUABYAkiQV6KLoBIZZAEiSVKBbgNXRSeAMgCRJhRoClkQngR0ASZIK99PoBLADIElS4X5K/O2AdgAkSSrYCmB5cA6TLQAkSSpe9DLAJAsASZKKd1Xw4w9aAEiSVLylxM4B2AGQJCnAY8DdgY9vASBJUpClgY/tEoAkSUEiCwA7AJIkBbkm8LEHBwIfXJKkJhsgnQswI+Cxn7MDIElSjCHS4UARXAKQJClQVAHgEKAkSYHsAEiS1EA3Bz3uGocAJUmKMwg8CWxV8OPeZQdAkqQ4G4GbAh53pQWAJEmxrgp4zJ8HPKYkSRrlANItgUXGPoU8M0mS1NYk4NcUd/G/j7QJkSRJCnYsxRUAHy/oOUmSpAlMpZguwN3AZgU9J0mS1IGFwDryu/ivA/Yv7NlIkqSO/Q35FQDHF/g8JElSl/6R7C/+Jxf5BCRJUm+OB56n/wv/euC4gnOXJEl92BNYRu8X/18AexWetSRJ6tsgcBRwPZ1d9DcCVwLHDP+/bbkRgCRJ1bAz8C7SzoFzgTmk2wdXAXeQugVLgPuD8pMkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIk1dD/B8A8fHJ9Y3ihAAAAAElFTkSuQmCC",
      "type": "command"
    },
    "ReviewAndShareCommand": {
      "title": {
        "default": "Review & Share"
      },
      "iconImageUrl": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAABuwAAAbsBOuzj4gAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAXnSURBVHic7ZtpjBVFEMd//2XDFS4xgEQFBBQCBuVQjIISjxB18WRF4kEQ/IQHRuNBQhY8uDQR+KJGxaByq6AoRqMmBmKEiIpuNCIaiUIMhF3lWFGO8kP3srO775id997M8pZKJpPpqq6u+U91dXXNjMyMlkwlSRuQNJ0CIGkDkqbSbAKSLgDOicGWINWY2cdxDJQRAEnXAmuBNnEYE6CdQJ84BkoLgKQy4C3czR8AlsRhEPBgTOMAoFTLoKQbgDW4GFEK7DSzPrEYJFmc4zUKgpJuwj35VsAdcRiRJNUDQNItwGrffqeZrU7EqhjpBACSyoFVuCc/ycxWJmZVjFQCIGkCsNxfTzazZYlaFSOVSpoIvIF78m8D+30caBEk4Cju5jNR0a4Cpbibz7bOV8VhTBJUmwhVmdn0RC1JiFr8ZqjFAxBmNxhpRTCzdWFlJXUG+gK9fVNbSd3NbE+UsZtKBvxmZqQ6PL/JRzp9Ab0dgYeBTcCxFDqOAZ8DDwBnZdMX9ZAfLO2yI2lhJFQzBFVJo4Gl1NUZDNgBfAXsAYYCw4AOAf47wBQz+zuKPRltJYMH5PMAWgPzqXvia4ArgS4pZEuAQcA0YJ+X3w6cH7cHhIoB2ea8pG7AJ8AQf0PTzGxVGN2SegIvAeOAQ8C9ZrYiTN8wlJcYEOLpr/GyG4AeET1oElADHAdG5cMDsq4CwKJcEZZ0GzAeN8/Hm1lNFD1mtlRSJ2Ax8KKkoWZ2JFf7ChoDgO7AXty8vzQP+gR85u1+LA/6Cg7Am36MBVnkOgHXALcDA/DlujSyvYH9uHjQq9kCALTDzdl/gLYp+F2AOUAljXOBfcB6YEwa3XO83P252FjoTPAqD8IWMzsc0FkKTAdmAKcFum0EvgPuAroCZUCZpA3AI2b2Y0B2sz9fGMW+erZSoEwQmOmvX2jg6h81kD8IXNJA5rUGMgeAsoDMmb59a0E9gNxWgTP8+WsASb1wy+Bg4CfgB+BmYKWZfVnbycz2S6oA7vZNz+PS5nclPWRmi81sl6Q/gcGSSs3saFQjCxkDVnn9o/x1V5ybfwh0BpZ5/nVp+m/x/P644FgFlAf4n3p+v8RjgKXOBKv9uR+wycyqJF0NHDGz45J+9vyxOM8IjtsTN78P4zLVHZLes/o5RH9cSW9XGBvT2k6BMkFglucvSsMf7fnVBJYzXJluoed9kKZvN8+vbM4xoNKfh6dimtlGSa8CU4BfJa3HrQKTgbNxwfG+NLpH+PO2HOxzdlC4GNAR58IHgVZpZNoAC2icB1QCwzLorvByj+ZoY8EzwbV+jMezyPUCJuAKIJcBrTPIdsfVDf4DBjR3AGrT1n/J014e9wLHgPm56spbPaDRvAqsCpKmAi8DW3EJT+Q127/JWg78AQw0s0NRdZ2wlRhqgrhlzoBXgHYRn/wY6qpD5VF0NDxiqQd4mopLbKYAoyVNNrMvwnSU1B6Yh1sR5JuH4YosOVNBY0CDJ9gFeJ26qu+zQN8M8h2A63GFlNr9wDzgL389N/EYYE2o/wd03oir8fXwTdW4inCwKjwCGEjdy5sVuB3hbkkjcRuqzsA8M3vC6+0P3Gpm85tiT04xIAdvOB14Dvglg/7dwPvAFSn6jyTgCbi0+Hd/PSefHpDxvYDl4aWqpEHAebgtbmvge2Cbme3N0i/oCQdxU+YYLpWea2YzwowfWwwoUFyZSJ3HLMFNnSp//UyI/icvANR3+yVAiW8fFgDh6ZMKAFyp7J4QciW4/YIB39TefIA/PAwIzRGA0PZQPxDOSsEfjlthDHiq6AAICcKIAAhPFh0AEUCYXXQAhAThogB/dqC9OAAICcLFAX5F0QEQAYRxRQdASBDKPW9zUQKQDQTcx2E1wIHaekD7Yvs+2Mw2SxqL2ytUSAIX/Az3WU474FuIWPEp8JE3j6S+J2wH1uGKqQZMKiV/FZ98Ut6+TfaecDnui/ghwLk4AGaa2dKU/wwVI0lqhSuwdMNttashzU9TLYla/LfCpwBI2oCk6X/OXU+9wUkXNgAAAABJRU5ErkJggg==",
      "type": "command"
    },
    "RecycleBinCommand": {
      "title": {
        "default": "Recycle bin"
      },
      "iconImageUrl": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBkPSJNNyA0YTIgMiAwIDAgMSAyLTJoNmEyIDIgMCAwIDEgMiAydjJoNGExIDEgMCAxIDEgMCAyaC0xLjA2OWwtLjg2NyAxMi4xNDJBMiAyIDAgMCAxIDE3LjA2OSAyMkg2LjkzYTIgMiAwIDAgMS0xLjk5NS0xLjg1OEw0LjA3IDhIM2ExIDEgMCAwIDEgMC0yaDRWNHptMiAyaDZWNEg5djJ6TTYuMDc0IDhsLjg1NyAxMkgxNy4wN2wuODU3LTEySDYuMDc0ek0xMCAxMGExIDEgMCAwIDEgMSAxdjZhMSAxIDAgMSAxLTIgMHYtNmExIDEgMCAwIDEgMS0xem00IDBhMSAxIDAgMCAxIDEgMXY2YTEgMSAwIDEgMS0yIDB2LTZhMSAxIDAgMCAxIDEtMXoiIGZpbGw9IiMwRDBEMEQiLz48L3N2Zz4=",
      "type": "command"
    },
    "AdobeCommand": {
      "title": {
        "default": "Adobe"
      },
      "iconImageUrl": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAfCAYAAACGVs+MAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAARGSURBVFhHvVdrbFNlGH7O6WVt1xXWsmFH2djGtnB1E2IikUAEUSCTYYiDDGNcJEoQDcl+EIn/BgmJF0ANEVRCgkAWE7klxp+EKAaZGBkMGCx0Fyczbh2W3k7POb7v124Zo3U7J7An+U7P952e732+5721EtJ4a9Pv/mhE3Ua363Qd82kQ+DLylTQyrY1G5ucSLdFoo3E612354stj1X1inS/1ay4vki04RUYDPH/SIBI9ZKvu5LnFrdIbG37zx2PaJVqfFOPDYBJOl+VZOanoTTSfVOMMVjsR15pkckJtes0QNE1HMqnzRqZBr9bKJEWF0U1UVUe+146y2S4xN0tC11Ah00VE6ETBxgPFTny4pxK7mivRuLWYCJhjwK+xCwyBYgbPL/fi+tV/cfDTu1i1tgCzylzCHWYgpz8NIc9jRV9vDOFwUsyLAg5opIxx6OYIhAYVeKfZMYMMM8TpDSqZgmScABUsdN6OoLTchXkL88Ta3/fikGUzDEwoYLFIuHk9TFlgw5JlXgwOKEQgYZKAGQXIUIiMtreFBZlb7WEMDSm0nv6CIZhQgFPHniNTKqb8z8FoNgMYhgmoZKy8woXKOW4xX/FyAYpLnCI9zcA4AUq3pS/4RPG6/EsInR0RbGsqhdUmCSV4KESGB5Mdr0YZIsC5Pq3QjueW5ov5D2f68VHzbXh9dmzZXoKp+TahzsIaD55+xoMSKlA2IqYkqNxmgdRQ16oz04kgFtOwYZMfDY0BdAejOHQgiDIyuOIlckOpE5GIip6uqHAHB6g7zyqUajn2J346PwCr9eFM4WcTIsD9Ih7X4CuwYe+BufRpF9LSLygM/JNA+7Uwgp0RrH5lOrruRvHtNz3o7Y4i123FxwfnId9nw/bGq6KAjU5XiRiMS4B6tii97PcX19BJZzlxry+O71v6cINSke/jJLFO7ZmrY/3rRaheNAVdpFAsqom+wYVq53vtVLrVh9L1fxXgfq+pwLKVPtS95ufTqrOrcvklS/OuW7j0cwgOpyxOxBsNv8Py+2c4RAwUzXQIlS5eGBQuY7eMRlYFRJskplveLUHVXDcOfx7EgmoPXt3ox4/n+nHosyDs9uzxy5nCew5nAGfIWOMMJpBxF0XRhLHCp3Kw4+02eMgFPL/T8QDHj/Q+Ekxjwca4WOU4UiOT8RSyVEJmNmd+Hv64cj+5am2hvuODcgqqGD7ZfUdEurm6nxlSwzpywZhezsWEc/nNd4q1KVOt0q8XQ9KJo72iB7CcjwscO9Lm9a26ktBHAmkYTMJB8tnI1/eHkrCQVnJWKc2BXcO/CTvGGmewn7mcPqDU4Wr2uI2nAz0oU+qcTa89AiZmrs2OD95bVfGd7HBa9tGkP70+aSCbf7ly5f3ykZaabvqzWE8nnTQSbJz8v/6rEzXdI45trL8SoP+I79NtLbmlarw2ahQpd0o36fYsVdD9X5+s6QGA/wA3Z8p+V98vSwAAAABJRU5ErkJggg==",
      "type": "command"
    }
  }
}
```

ListActionsCommandSet.ts
```Typescript
import {
  BaseListViewCommandSet,
  Command,
  IListViewCommandSetExecuteEventParameters,
  ListViewStateChangedEventArgs,
} from "@microsoft/sp-listview-extensibility";
import './ListActionsCommandSet.module.scss';
import { constants } from "@Constant/AppConstant";

import { SPFI, spfi, SPFx as spSPFx } from "@pnp/sp";
import { GraphFI, graphfi, SPFx as graphSPFx } from "@pnp/graph";
import "@pnp/sp/webs";
import "@pnp/sp/site-users/web";
import * as Actions from '@Store/Actions';
import { IRenderManipulation, } from "@Services/RenderManipulation/IRenderManipulationService";
import { RenderManipulation } from "@Services/RenderManipulation/RenderManipulationService";
import { cloneDeep } from '@microsoft/sp-lodash-subset';
import { IReviewService } from "@Services/Review/IReviewService";
import { ReviewService } from "@Services/Review/ReviewService";
import { UpdateMetadata } from "@Services/updateMetadata/UpdateMetadata";
import { IUpdateMetadata } from "@Services/updateMetadata/IUpdateMetadata";

import { indexList } from "@Helpers/indexList";
import AdobeDialog from "./component/Dialogs/AdobeDialog/AdobeDialog";
/**
 * If your command set uses the ClientSideComponentProperties JSON input,
 * it will be deserialized into the BaseExtension.properties object.
 * You can define an interface to describe it.
 */
export interface IListActionsCommandSetProperties { }

export default class ListActionsCommandSet extends BaseListViewCommandSet<IListActionsCommandSetProperties> {
  private _sp: SPFI;
  private _graph: GraphFI;
  private _highlightFolderService: IRenderManipulation;
  private _reviewService: IReviewService;
  private _updateMetadataService: IUpdateMetadata;
  private _userCanCreateItems: boolean;

  public onInit = (): Promise<void> => {

    this.context.serviceScope.whenFinished(() => {
      this._highlightFolderService = this.context.serviceScope.consume(RenderManipulation.servicekey);
      this._reviewService = this.context.serviceScope.consume(ReviewService.servicekey);
      this._updateMetadataService = this.context.serviceScope.consume(UpdateMetadata.servicekey);
    });

    Actions.SetWebPartContext(cloneDeep(this.context));
    this._graph = graphfi().using(graphSPFx(this.context));
    this._sp = spfi().using(spSPFx(this.context));

    Actions.SetSP(this._sp);
    Actions.SetGraphSP(this._graph);

    // initial state of the command's visibility
    const updateMetadataCommand: Command = this.tryGetCommand("UpdateMetadataCommand");
    updateMetadataCommand.visible = false;

    const goToIndexCommand: Command = this.tryGetCommand("GoToIndexCommand");
    goToIndexCommand.visible = window.frameElement ? false : true;

    const reviewAndShareCommand: Command = this.tryGetCommand("ReviewAndShareCommand");
    reviewAndShareCommand.visible = false;

    const recycleBinCommand: Command = this.tryGetCommand("RecycleBinCommand");
    recycleBinCommand.visible = true;

    const adobeCommand: Command = this.tryGetCommand("AdobeCommand");
    adobeCommand.visible = false;

    return super.onInit().then(async () => {
      constants.setupEnvironment(this.context);
      this.context.listView.listViewStateChangedEvent.add(this, this._onListViewStateChanged);

      // Create the container for approval panel
      const libraryName: string = this.context.pageContext.list.title;

      indexList.userCanCreateItems(libraryName).then((userCanCreateItems) => {
        this._userCanCreateItems = userCanCreateItems;
        updateMetadataCommand.visible = userCanCreateItems;
        // adobeCommand.visible = userCanCreateItems;
        this.raiseOnChange();
      }).catch((error) => {
        console.log(error);
      });

      this._highlightFolderService.highlightFolderRow(this.context).catch((error) => {
        console.log(error);
      });
    });
  }

  private _onListViewStateChanged = async (args: ListViewStateChangedEventArgs): Promise<void> => {

    const updateMetadataCommand: Command = this.tryGetCommand("UpdateMetadataCommand");
    const goToIndexCommand: Command = this.tryGetCommand("GoToIndexCommand");
    const reviewAndShareCommand: Command = this.tryGetCommand("ReviewAndShareCommand");
    const recycleBinCommand: Command = this.tryGetCommand("RecycleBinCommand");
    const adobeCommand: Command = this.tryGetCommand("AdobeCommand");

    if (updateMetadataCommand) {
      updateMetadataCommand.visible = this._userCanCreateItems;
    }
    if (goToIndexCommand) {
      goToIndexCommand.visible = window.frameElement ? false : true;
    }
    if (recycleBinCommand) {
      recycleBinCommand.visible = true;
    }
    if (reviewAndShareCommand) {
      if (this._userCanCreateItems) {
        const selectedRows = this.context.listView.selectedRows;
        if (selectedRows?.length) {
          reviewAndShareCommand.visible = selectedRows.some(i => (i.getValueByName("FSObjType") === "0"));
        } else {
          reviewAndShareCommand.visible = false;
        }
      }
    }
    if (adobeCommand) {
      adobeCommand.visible = false; // hide adobe command
      // const selectedRows = this.context.listView.selectedRows;
      // In Development
      // if (selectedRows?.length === 1) {
      //   adobeCommand.visible = selectedRows.some(i => (i.getValueByName("FSObjType") === "0"));
      // } else {
      //   adobeCommand.visible = false;
      // }
    }

    /* highlight core folder rows */
    this._highlightFolderService.highlightFolderRow(this.context).catch((error) => {
      console.log(error);
    });
    // You should call this.raiseOnChange() to update the command bar
    this.raiseOnChange();
  };

  public onExecute = async (event: IListViewCommandSetExecuteEventParameters): Promise<void> => {

    const libraryName: string = this.context.pageContext.list.title;
    const siteURL: string = this.context.pageContext.site.absoluteUrl;

    switch (event.itemId) {
      case "UpdateMetadataCommand":
        await this._updateMetadataService.update(libraryName);
        break;
      case "GoToIndexCommand":
        this._redirectToIndexList(libraryName);
        break;
      case "ReviewAndShareCommand":
        {
          const selectedItems = event.selectedRows;
          if (selectedItems.length) {
            await this._reviewService.review(selectedItems, libraryName);
          }
        }
        break;
      case "RecycleBinCommand":
        this._redirectToRecycleBin(siteURL);
        break;
      case "AdobeCommand": {
        const dialog: AdobeDialog = new AdobeDialog({
          isBlocking: true
        });
        dialog.context = this.context;
        dialog.documentId = event.selectedRows[0].getValueByName("fileUniqueId");
        dialog.documentPath = event.selectedRows[0].getValueByName("FileDirRef");
        dialog.documentName = event.selectedRows[0].getValueByName("FileLeafRef");
        await dialog.show();
        break;
      }
      default:
        throw new Error("Unknown command");
    }
  };


  private _redirectToIndexList = (filter: string): void => {
    try {
      const listLink: string = constants.indexListLink.replace("##ID##", filter);
      const url: string = constants.indexSiteUrl + listLink;
      window.location.assign(url);
    } catch (error) {
      console.log(error);
    }
  };
  private _redirectToRecycleBin = (filter: string): void => {
    try {
      const urlAux: string = "/_layouts/15/RecycleBin.aspx";
      const recycleBinURL: string = filter + urlAux;
      window.open(recycleBinURL, '_blank').focus()
    } catch (error) {
      console.log(error);
    }
  };

}
```
Note:
The "Actions" is a store were we keep the values to be used anywhere in the code.

**Comment by natalieethell** [comment]
Thanks @Dangerous-Mind . Do you see any console logs or errors in the ?env=WebViewList case? Also, is this an issue for your command sets in both versions 1.19 and 1.13?

**Comment by Dangerous-Mind** [comment]
@natalieethell 
Sorry for the delay, too much work.
I have no errors on the console.

But I have the same Command set installed in different ways in several tenants in different Geolocations.

1. Geolocation 1 Solution Installed in the Global app catalog then installed on some of the sites manually:
- [x] Without any parameters
- [x] ?env=Embedded
- [ ] ?env=WebViewList
- [ ] ?env=WebView

2. Geolocation 2 Solution Installed in the Global app catalog then installed on some of the sites manually:
- [x] Without any parameters
- [x] ?env=Embedded
- [x] ?env=WebViewList
- [x] ?env=WebView

3. Geolocation 3 Solution Installed in the Global app catalog then installed on some of the sites manually:
- [x] Without any parameters
- [x] ?env=Embedded
- [x] ?env=WebViewList
- [x] ?env=WebView

4. Geolocation 1 Installed in local app catalog (Global deployed to this Site collection)
- [x] Without any parameters
- [x] ?env=Embedded
- [x] ?env=WebViewList
- [x] ?env=WebView

5. Geolocation 2 Installed in local app catalog (Global deployed to this Site collection)
- [x] Without any parameters
- [x] ?env=Embedded
- [x] ?env=WebViewList
- [x] ?env=WebView

6. Geolocation 3 Installed in local app catalog (Global deployed to this Site collection)
- [x] Without any parameters
- [x] ?env=Embedded
- [x] ?env=WebViewList
- [x] ?env=WebView

7. Another Tenant on Geolocation 1. Installed in local app catalog (Global deployed to this Site collection)
- [x] Without any parameters
- [x] ?env=Embedded
- [x] ?env=WebViewList
- [x] ?env=WebView

8. Personal Tenant on Geolocation 1. works Globally deployed or local installed locally

So, I can't conclude anything. It seams to happen only on one European (Geolocation 1) tenant and in several site collections.

**Comment by natalieethell** [comment]
Hi @Dangerous-Mind I'm sorry that this fell off my radar. How are customizers working for you today? Are you still having the same issue?

**Comment by Dangerous-Mind** [comment]
Hi @natalieethell ,

I didn't had this problem on most of my installations and didn't received any complaints about this issue.
So, it's probably solved. 

But I have now another problem involving the events of the new command set version. 

When the user changes to another folder, it doesn't trigger an event with the new context. 
The new context event is only triggered if something is selected on the new folder. 

But if you set a sleep timer for some seconds, the context was changed showing the new folder.

We already reported that and are waiting for a fix.

Regarding this one, I think you can close it but it's better to confirm with @JLRishe .
